ui/app/AppLayouts/Chat/ChatColumn/ChatMessages.qml

```diff
@@ -56,17 +56,30 @@ Item {
             }
         }
 
-        property var scrollToMessage: function (messageId) {
-            let item
-            for (let i = 0; i < messageListDelegate.count; i++) {
-                item = messageListDelegate.items.get(i)
-                if (item.model.messageId === messageId) {
-                    chatLogView.positionViewAtIndex(i, ListView.Center);
-                    if (appSettings.useCompactMode) {
-                        chatLogView.itemAtIndex(i).startMessageFoundAnimation();
+        property var scrollToMessage: function (messageId, isSearch = false) {
+            delayPositioningViewTimer.msgId = messageId;
+            delayPositioningViewTimer.isSearch = isSearch;
+            delayPositioningViewTimer.restart();
+        }
+
+        Timer {
+            id: delayPositioningViewTimer
+            interval: 300
+            property string msgId
+            property bool isSearch
+            onTriggered: {
+                let item
+                for (let i = 0; i < messageListDelegate.count; i++) {
+                    item = messageListDelegate.items.get(i)
+                    if (item.model.messageId === msgId) {
+                        chatLogView.positionViewAtIndex(i, ListView.Center);
+                        if (appSettings.useCompactMode && isSearch) {
```

Comment:
if its not in `compactMode` no need to show animation?

Reply:
There is no definition for NormalMessage, which btw has no highlight either and looks different.